.factory/rules/ox.md

```
@@ -0,0 +1,48 @@
+| List teams | `ox teams` |
+| Load expert coworker agent | `ox coworker load <name>` |
+
+## Session Recording
+
+Sessions auto-record when `ox agent prime` runs. Discussions may be shared with teammates.
+
+## Murmuring
+
+Publish WIP to teammates so they stay in sync:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)